Core/Src/logic_layer.c

```diff
@@ -49,40 +124,40 @@ void logic_layer(char commando[])
 						(uint16_t)atoi((char*)commando_filled[2]),
 						(uint16_t)atoi((char*)commando_filled[3]),
 						(uint16_t)atoi((char*)commando_filled[4]),
-						(uint16_t)atoi((char*)commando_filled[5]),
+						(uint16_t)kleur_decoder((char*)commando_filled[5]),
 						(uint16_t)atoi((char*)commando_filled[6]));
 				break;
 			case 1:
 				drawRect((uint16_t)atoi((char*)commando_filled[1]),
 						(uint16_t)atoi((char*)commando_filled[2]),
 						(uint16_t)atoi((char*)commando_filled[3]),
 						(uint16_t)atoi((char*)commando_filled[4]),
-						(uint16_t)atoi((char*)commando_filled[5]),
+						(uint16_t)kleur_decoder((char*)commando_filled[5]),
 						(uint16_t)atoi((char*)commando_filled[6]));
 				break;
-			/*case 2:
-				drawLines(100,10,200,10,0x1F,5);
-				/*drawText((uint16_t)atoi((char*)commando_filled[1]),
+			case 2:
+//				drawLines(100,10,200,10,0x1F,5);
```

Comment:
delete this line!

Reply:
deleted

Reply:
![LiarLiarLiarMovieGIF](https://user-images.githubusercontent.com/46404391/169264235-f0d1bbc1-6651-4695-a8c5-d06f8df4dbc6.gif)